rcv/rcv_server.py

```diff
@@ -14,6 +14,7 @@ def exit_gracefully(signum, frame):
 
 
 signal.signal(signal.SIGTERM, exit_gracefully)
+signal.signal(signal.SIGINT, exit_gracefully)
```

Comment:
why did you add SIGINT ?

Reply:
because when i used ctrl c it didn't stop my program and i wanted to close it